### DISCUSSION TITLE
SOF version 1.3 and release cadence

### DISCUSSION THREAD

**Discussion by lgirdwood** [thesofproject/sof#7372] [Team Posts]
Ist item of business is to discuss SOF 1.3 FW release and FW release cadence. This was delayed due to my accident. Is there anything outstanding feature/issue wise that will prevent an RC1 next week ?

I would like to do a quarterly or eight weekly release cycle. Any opinions on this ?

**Comment by plbossart** [comment]
releasing now is a bit odd, do you want to include hotwording stuff or not?

**Comment by lgirdwood** [comment]
I was thinking hot wording and IMX support for v1.4 about 8 weeks after 1.3 ? Just to let things settle on the KD side.... My hope for 1.3 was stable HDA/HDMI and PM.

**Comment by dbaluta** [comment]
8 weeks after 1.3 should be doable for IMX.

**Comment by lgirdwood** [comment]
You should have a bit longer give the regression with BYT and Pulseaudio.
